bin/upstream-validator/schema.json

```diff
@@ -0,0 +1,68 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://smarthealthit.org/slot.publisher.schema.json",
+  "title": "Vaccine Slot Publisher Endpoint",
+  "description": "",
+  "type": "object",
+  "properties": {
```

Comment:
do you think we could unify this data model with the JSON-LD data folks can publish on their website, so there's a clear equivalence, and the same content can be validated either way?

Reply:
It might need a service + some other data or ad-hoc properties, but should be doable.

Reply:
That's a good point, I think we can do that!

Reply:
I'm not sure I know enough about schema.org to do this easily.

Could you put together an example JSON-LD unifying the fields? I can then update the docs and generate the schema for it.

Reply:
I'm not sure I do either, but here's my best take :-)

```
<script type="application/ld+json">
{
    "@context": "https://schema.org/", 
    "@type": "Service", 
    "name": "SMART Example Slot Publisher", 
    "serviceType": "https://github.com/smart-on-fhir/smart-scheduling-links", 
    "category": "production | testing", 
    "url": "https://api.carbonhealth.com/hib/publicVaccination/$bulk-publish", 
    "provider": {
        "@type": "Organization", 
        "name": "SMART Health IT", 
        "contactPoint": [
            {
                "@type": "ContactPoint", 
                "name": "Nick Robison", 
                "email": "nicholas.a.robison@omb.eop.gov"
            }, 
            {
                "@type": "ContactPoint", 
                "name": "Josh Mandel", 
                "email": "jmandel@med.harvard.edu"
            }
        ]
    }, 
    "additionalProperty": [
        {
            "@type": "PropertyValue", 
            "additionalType": "ApiHeader", 
            "name": "CLIENT_ID", 
            "value": "test-client-id"
        }
    ]
}
</script>
```

Reply:
Thanks! So how would we represent the `synthetic` value? I'm wondering if this might be too complicated for the GitHub side of things. I like the idea of keeping things simple and easy to input. Feels like there's a lot of extra information and context here.

Reply:
I was expecting the `category` would say production or testing; while "synthetic" might be one kind of testing data, there could be other kinds of testing data as well, and I'm not sure that it's important to differentiate (e.g.,  you might also have real world testing data that are just old or limited in scope).

I think we could validate this entire payload using json schema, just as for the simpler example. I agree there is a little bit of extra boilerplate here, but the advantage is that providers only need to create one snippet of json, whether they are submitting it to the registry or publishing it on their own website. Otherwise they need to craft two different payloads with effectively the same information.